staging/src/k8s.io/legacy-cloud-providers/gce/gce.go

```diff
@@ -542,6 +529,45 @@ func CreateGCECloud(config *CloudConfig) (*Cloud, error) {
 	return gce, nil
 }
 
+// initializeNetworkConfig() is supposed to be called under sync.Once()
+// for accessors to subnetworkURL and isLegacyNetwork fields.
+func (g *Cloud) initializeSubnetworkURLAndIsLegacyNetwork() {
+	if g.subnetworkURL != "" {
```

Comment:
Is this correct check if network type is Legacy or Custom?
I think in that case g.subnetworkURL is expected to be empty.

Reply:
See the line 475 - if subnet was provided via config it already had been initialized before - I only moved the "else" part to here.
Basically - it could already have been initialized 470-474 and if it was it return from here, otherwise call the previous else.

This should be no-op change.